model/vcs/metrics.yaml

```diff
@@ -0,0 +1,139 @@
+groups:
+  - id: metric.vcs.repository.change.count
+    type: metric
+    metric_name: vcs.repository.change.count
+    brief: 'The number of changes (pull requests) in a repository, categorized by their state (either open or merged)'
+    instrument: updowncounter
+    unit: "{change}"
+    stability: experimental
+    attributes:
+      - ref: vcs.change.state
+        requirement_level: required
+      - ref: vcs.repository.url.full
+        requirement_level: required
+  - id: metric.vcs.repository.change.time
+    type: metric
+    metric_name: vcs.repository.change.time
+    brief: 'The time distribution a change (pull request) has been in open state or that it took to go from open to another state.'
```

Comment:
```suggestion
    brief: 'The time duration a change (pull request/merge request) has been in a given state.'
```

Reply:
Done in cfcdaf1ac3a1887af58cdcadb40863d660b64a82

Reply:
I also added `/merge request` to all other metric briefs relating to changes in e174ef3572be947aa46d638168c8c287e6e32536.